README.md

```diff
@@ -170,14 +170,6 @@ The following configuration has been renamed, the previous keys are invalid and
 | httpHeadersWhitelist     | bootstrap.http.headersAllowlist    |
 | csrfexceptions.whitelist | bootstrap.csrfexceptions.allowlist |
 
-### Version 0.37.0
-
-#### Play auditing
-play-auditing was bump in major version - changes in [`HttpAuditing#auditFromPlayFrontend`](https://github.com/hmrc/play-auditing/blob/be52847234b113eab713faaeb6b40d00bf4a3bcd/src/main/scala/uk/gov/hmrc/play/audit/http/HttpAuditing.scala#L52) signature  were made.
```

Comment:
This method has been removed as it's not used from anywhere. These outdated comments don't add anything